src/feed/TickHistoryFormatter.js

```diff
@@ -1,16 +1,21 @@
 export class TickHistoryFormatter {
+    static getUTCDate(epoch) {
+        const UTCdate = new Date(epoch).toISOString();
+        return UTCdate.substring(0, 19);
+    }
+
     static formatHistory(response) {
         const { history, candles } = response;
         if (history) {
             const { times, prices } = history;
             const quotes = prices.map((p, idx) => ({
-                DT: +times[idx] * 1000,
+                Date: this.getUTCDate(+times[idx] * 1000),
```

Comment:
epoch is in seconds. Do the `* 1000` conversion in `getUTCDate`

Reply:
@brucebinary yes right.